interview-questions.md

```diff
@@ -7,65 +7,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: MVC stands for model view controller and it is a software design pattern. I know it breaks things into three components
 
-  Researched answer:
+  Researched answer: It seems that it is a design that divides the related logic into three elements which are the model view and controller. The user sees the view and uses the controller which manipulates the model which when updated changes the view which the uses then sees.
 
 
 
 2. What does it mean if an application has CRUD? What does CRUD stand for?
 
-  Your answer:
+  Your answer: CRUD stands for create, read, update, and delete which are the requirements for a full-stack application
 
-  Researched answer:
+  Researched answer: Pretty much what my answer said but these four functions are considered necessary to implement a persistent storage application.
 
 
 
 3. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: Only know that it will most likely have to do with relationship between two different data bases.
 
-  Researched answer:
+  Researched answer: A relational database is a collection of information that organizes data points with defined relationships for easy access. It is also the most widely used database model. The main component is the access to data points that are related to one another.
 
 
 
 4. What are the 5 HTTP verbs? What are they important?
 
-  Your answer:
+  Your answer: I am not sure at all what they are
 
-  Researched answer:
+  Researched answer: They are POST, GET, PUT, PATCH, and DELETE. These are important because they are the same as the verbs that are in CRUD.
 
 
 
 5. What is object-relational mapping?
 
-  Your answer:
+  Your answer: I know it must do with objects oriented programming but not sure where the relational part comes into play.
 
-  Researched answer:
+  Researched answer: From my understanding it is the use of enwrapped software that translate and manipulates information from a relational database table into an object in Ruby by using ruby on rails
 
 
 
 6. What is a gem?
 
-  Your answer:
+  Your answer: I know that it has to do with Ruby and is like a an addon or plugin that can help extend rubys capabilities.
```

Comment:
This is great. Indicating what you know about a thing even when you don't know what the thing is. Keep this habit up. It'll be helpful in interviews and in software in general!